tests/performance/local/libcds_hp_overhead.cpp

```diff
@@ -0,0 +1,345 @@
+//  Copyright (c) 2020 Weile Wei
+//  Copyright (c) 2020 John Biddiscombe
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0.
+//  (See accompanying file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+
+#include <cds/gc/hp.h>    // for cds::HP (Hazard Pointer) SMR
+#include <cds/init.h>     // for cds::Initialize and cds::Terminate
```

Comment:
We usually put external headers after HPX headers and before standard library headers.